cluster-autoscaler/simulator/clustersnapshot/clustersnapshot.go

```diff
@@ -43,6 +44,16 @@ type ClusterSnapshot interface {
 	// IsPVCUsedByPods returns if the pvc is used by any pod, key = <namespace>/<pvc_name>
 	IsPVCUsedByPods(key string) bool
 
+	// AddNodeInfo adds the given NodeInfo to the snapshot. The Node and the Pods are added, as well as
+	// any DRA objects passed along them.
```

Comment:
rather than refer to "DRA objects" generally, should we be more explicit and refer to "ResourceClaims" and "ResourceSlices"?

Reply:
I just tried to make the comment generic enough so it doesn't get outdated if we there are more DRA objects to track in the future.